modules/oe_theme_content_publication/oe_theme_content_publication.post_update.php

```diff
@@ -153,3 +153,23 @@ function oe_theme_content_publication_post_update_00008(): void {
   $display = $storage->createFromStorageRecord($display_values);
   $display->save();
 }
+
+/**
+ * Set labels in teaser view mode to be hidden.
+ */
+function oe_theme_content_publication_post_update_00009() {
+  $display = EntityViewDisplay::load('node.oe_publication.teaser');
```

Comment:
We cannot assume that the teaser is still there on a site using this module, let's first check that it exists.